[DOCS] Update patching options and daal4py install guide (#880)
intel/scikit-learn-intelex: README.md

Intel(R) Extension for Scikit-learn offers you a way to accelerate existing scikit-learn code.
The acceleration is achieved through **patching**: replacing the stock scikit-learn algorithms with their optimized versions provided by the extension.

> One of the ways to patch scikit-learn is by modifying the code. You import an additional Python package (`sklearnex`) and enable optimizations via `sklearnex.patch_sklearn()` (Pay attention: need to import scikit-learn estimators after patching lines):

Reviewer: ```suggestion
One of the ways to patch scikit-learn is by modifying the code. First, you import an additional Python package (`sklearnex`) and enable optimizations via `sklearnex.patch_sklearn()`. Then import scikit-learn estimators:
```
